web/README.md

```diff
@@ -35,6 +35,26 @@ running Agama server instance. This is especially useful if you use the Live ISO
 which does not contain any development tools, you can develop the web frontend
 easily from your workstation.
 
+Example of running from different machine:
+
+```
+  # backend machine
+  # using ip of machine instead of localhost is important to be network accessible
+  agama-web-server serve --address 10.100.1.1:3030
+
+  # frontend machine
+  # ESLINT=0 is useful during development when there is some styling issues
+  ESLINT=0 AGAMA_SERVER=10.100.1.1:3000 npm run server
+```
+
+### Debugging Hints
+
+There are more locations where to look when something does not work and require debugging.
+The first location to check when something does not work is browser console which can give
+some hints. The second location to check for errors or warnings is output of `npm run server`
+where some issues when communicating with backend can be seen. And last but on least is
+journal on backend machine where is logged backend activity `journalctl -b`.
```

Comment:
Thanks a lot for documenting this. I would add: `busctl monitor --address /run/agama/bus` as another source of information.

Reply:
well, I think that this busctl monitor and other useful tools for dbus observing belongs more to backend documentation. Either service or rust dir. Here I try to focus mostly on debugging issues with web UI.

Reply:
Yes, sure, but given that you mentioned `journalctl` too, I think it might be useful.

Reply:
well, I mention journalctl mainly because you see there if request from web UI lands to backend. If it is correctly send there, then I would continue with backend debugging. But maybe we can link backend debugging document from this one.

Reply:
Well, I would avoid having different documents about debugging. All the information about that matter should be in a single place.

Reply:
ok, so lets add it.